packages/components/src/styles/ProgressBar.module.css

```diff
@@ -9,16 +9,52 @@
 }
 
 .progress {
-	--fill: var(--lp-color-gray-500);
+}
+
+.spinner {
+	--fill-color: var(--lp-color-gray-500);
 	display: inline-flex;
 }
 
+.bar {
+	--fill-color: var(--lp-color-brand-cyan-base);
+	display: flex;
+	flex-direction: column;
+	gap: var(--lp-spacing-300);
+}
+
 .outerCircle {
 	stroke: var(--lp-color-white-100);
 }
 
 .innerCircle {
-	stroke: var(--fill);
+	stroke: var(--fill-color);
+}
+
+.container {
+	display: flex;
+}
+
+.label {
```

Comment:
Do we need this? Looks like we encourage passing the `<Label/>` component in and that should already be styled to be consistent with other labels.

Reply:
I guessed semibold from the design shared but we can just use the standard label.